patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -108,12 +108,16 @@
 +   @Nullable
 +   @Override
 +   public net.minecraftforge.common.ToolType getHarvestTool(BlockState state) {
++      if (this.harvestTool == null && net.minecraft.item.AxeItem.field_234662_c_.contains(state.func_185904_a()))
++         return net.minecraftforge.common.ToolType.AXE;
 +      return harvestTool; //TODO: RE-Evaluate
 +   }
 +
 +   @Override
 +   public int getHarvestLevel(BlockState state) {
-+     return harvestLevel; //TODO: RE-Evaluate
++      if (this.harvestLevel < 0 && net.minecraft.item.AxeItem.field_234662_c_.contains(state.func_185904_a()))
```

Comment:
This logic is just wrong.. it doesn't take into account levels that are already set.
However, this does look like getHarvestLevel is wrong, the javadocs say it should have a tool class passed in. Perhaps this got lost somewhere in the shuffle. We should re-add it.

Reply:
The check `this.harvestLevel < 0 &&...` takes that into account, I think. Or what do you mean? 
I could re-add the tool type to this method, if you want. :)